Mage.Tests/src/test/java/org/mage/test/cards/single/ncc/ThreefoldSignalTest.java

```diff
@@ -101,4 +101,76 @@ public void testShouldNotWorkOpponent() {
         execute();
         assertPermanentCount(playerB, esperSojourners, 1);
     }
+
+    /**
+     * Check that casting one half of a split card doesn't trigger it even if the whole split card has 3 colors.
+     * Relevant ruling:
+     *      709.3a Only the chosen half is evaluated to see if it can be cast.
+     *             Only that half is considered to be put onto the stack.
+     *      709.3b While on the stack, only the characteristics of the half being cast exist.
+     *             The other half’s characteristics are treated as though they didn’t exist.
+     */
+    @Test
+    public void oneHalfOfSplitCardDoesntTrigger() {
+        // {G}{U} / {4}{W}{U}
+        // Beck: Whenever a creature enters the battlefield this turn, you may draw a card.
+        // Call: Create four 1/1 white Bird creature tokens with flying.
+        // Fuse
+        String beckCall = "Beck // Call";
+        addCard(Zone.HAND, playerA, beckCall);
+
+        addCard(Zone.BATTLEFIELD, playerA, threefoldSignal);
+        addCard(Zone.BATTLEFIELD, playerA, "Island", 2);
+        addCard(Zone.BATTLEFIELD, playerA, "Forest", 1);
+        addCard(Zone.BATTLEFIELD, playerA, "Plains", 2);
+        addCard(Zone.BATTLEFIELD, playerA, "Mountain", 4 + 3); // For generic costs and to have enough for replicate
+
+        setStrictChooseMode(true);
+        castSpell(1, PhaseStep.PRECOMBAT_MAIN, playerA, "Call");
+
+        setStopAt(1, PhaseStep.POSTCOMBAT_MAIN);
+        execute();
+
+        assertPermanentCount(playerA, "Bird Token", 4);
+    }
+
+    /**
+     * Test that casting a split card with fuse triggers if both halves together have 3 colors
+     */
+    @Test
+    public void fusedSplitCardTriggers() {
```

Comment:
Strangely, every time replicate resolves, it will create 8 birds instead of 4. If I pay the replicate cost twice, I end up with 20 birds (4 original, 8 first replicate, 8 second replicate)

Reply:
Trying to use [[Flesh // Blood]] doesn't fare better. 

In this case, the left hand side of the card does not get copied properly.
I am not asked if I wish to change the target card in the graveyard. I think the target card isn't getting copied properly since the copy apparently fizzles (no card is exiled and the target creature does not get any +1/+1 counters.
The right hand side appears to be copied properly since the target creature will deal damage twice.

Reply:
What happens if you copy a fused spell with something simple like [[Reverberate]]? Copying fused spells seems like the kind of thing that could be completely broken in xmage without anyone noticing (or with the fact that it doesn't work hidden away in one of the many ancient "feedback from [forum]" issues).

Reply:
> What happens if you copy a fused spell with something simple like [[Reverberate]]? Does that even work? Copying fused spells seems like the kind of thing that could be completely broken in xmage without anyone noticing (or with the fact that it doesn't work hidden away in one of the many ancient "feedback from [forum]" issues).

There are othered fuse spell issues throughout those tracking sets, but not about copying.

Copying with reverberate also doesn't work.

The issue for `Beck // Call` comes from here:
https://github.com/magefree/mage/blob/a94c837ad0cf9d6a544fc14a93563194c5ea7b54/Mage/src/main/java/mage/game/stack/Spell.java#L810-L820

It skips past the first spell (in a round about way) and then adds a copy of all of the other spell abilities. What this means is that `Call` gets added twice.

Reply:
I will look into the spell copying issue since I'm already in the middle of fixing a bunch of issues with splicing (which xmage implements in a way that's closely related to how it implements fuse). In exchange, can you look at https://github.com/magefree/mage/issues/9533? I'm about 90% sure it's a regression caused by you.